messaging/src/main/java/org/axonframework/commandhandling/gateway/ExponentialBackOffIntervalRetryScheduler.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.commandhandling.gateway;
+
+import org.axonframework.commandhandling.CommandMessage;
+
+import java.util.List;
+
+import static org.axonframework.common.BuilderUtils.assertStrictPositive;
+
+/**
+ * A RetryScheduler that uses a backoff strategy, retrying commands at increasing intervals when they fail because of
+ * an exception that is not explicitly non-transient. Checked exceptions are considered non-transient and will not
```

Comment:
trying to remove the double negative here
```suggestion
 * an exception that is possibly transient. Checked exceptions are considered non-transient and will not
```

Reply:
I just copied that text from Allard's original, but "not explicitly non-transient" has a different meaning than "possibly transient".